woocommerce/woocommerce-gateway-stripe
# feat: add manual capture notice

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1956

Refactored so that the "manual" vs "automatic" capture checks are consolidated to a class method.

With manual capture:
![Screen Shot 2021-09-30 at 2 55 46 PM](https://user-images.githubusercontent.com/273592/135522029-5ffe149e-5c26-4b16-ad3c-49ded658ae1b.png)

With automatic capture:
![Screen Shot 2021-09-30 at 2 56 09 PM](https://user-images.githubusercontent.com/273592/135522059-56a7974d-0936-4d24-84d0-2fa0c65368de.png)

# Testing instructions
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Enable UPE
- Disable the "Capture charge immediately" checkbox
- Save the settings
- Notice should appear on the payment methods table
